frontend/src/components/molecules/ColumnMenuFilter/index.jsx

```diff
@@ -13,6 +13,13 @@ import TableColumnIcon from "@/components/atoms/TableColumnIcon";
 import { uid } from "@/utils/helper";
 import { useSelector } from "react-redux";
 
+/**
+ * @param {*} props
```

Comment:
How about a paragraph describing what the component *does*, as you did elsewhere? (E.g., `TableFilters`.)